client/packages/common/src/authentication/api/api.ts

```diff
@@ -79,10 +79,15 @@ export const getAuthQueries = (sdk: Sdk, t: TypedTFunction<LocaleKey>) => ({
         if ('message' in error) {
           console.error(error.message);
         }
+
+        const errorMessage = error.message.includes('Network request failed')
```

Comment:
I think this could be typed, through GraphqlStdError. Not suggesting we should do it thought, just food for thought